source/features/sticky-sidebar.css

```diff
@@ -4,6 +4,9 @@
 
 .rgh-sticky-sidebar {
 	position: sticky;
+}
+
+:not(#partial-discussion-sidebar).rgh-sticky-sidebar {
```

Comment:
Is this hunk needed at all? It looks like it will just be overridden by the next selector.

```suggestion
```

I think this is ready to merge after this change

Reply:
Yeah I split it that way for clarity's sake, but the selector with `:not()` is iffy